app/views/api/guidance/guidance_for_lead_providers/partnerships.md

```diff
@@ -0,0 +1,224 @@
+
+This reduces the need for lead providers to manually resubmit partnerships each cohort. 
+
+## Roll over partnerships 
+
+Schools can roll over their most recent partnership (not only from the previous cohort). 
+
+## Multiple partnerships 
+
+Previously, schools had a default partnership with a single lead provider for each cohort. 
+
+We’ve changed this for the 2026 cohort to reflect that some schools need to set up partnerships with more than one lead provider. 
+
+The API now supports multiple partnerships for a school in the same cohort: 
+
+* each lead provider can create one partnership per school per cohort
+* different providers can have partnerships with the same school (for example, if an ECT transfers and stays with their previous provider)
+* if a school needs several delivery partners under the same provider in the same cohort, lead providers must notify us (via Teams or email). We’ll then add these additional partnerships manually 
+
+In the API: 
+
+* `GET /partnerships` may return more than one partnership for the same school and cohort
+* `in_partnership` will state `TRUE` if any partnership exists and `FALSE` if none exist
+
+## Find schools and delivery partners 
+
+### Find schools delivering training 
+
+``` 
+GET /schools?filter[cohort]=year 
+``` 
+
+This endpoint returns schools eligible for funded training in a given cohort. Lead providers can check details on the type of training programme schools have chosen to deliver, and whether they’ve got confirmed partnerships in place. 
+
+Requests must include: 
+
+* `cohort` 
+
+Lead providers can also filter by URN. For example, `GET /schools?filter[cohort]=2024&filter[urn]=123456`. 
+
+For more detailed technical information, view the `GET /schools` [endpoint Swagger API documentation](/api/docs/v3#/Schools/get_api_v3_schools). 
+
+### What the API will show 
+ 
+The API shows schools eligible for funded training in a given cohort or schools with existing partnerships which have become ineligible.  
+
+### What the API will not show 
+ 
+The API will not show schools that are ineligible for funding in a given cohort. 
+ 
+If a school’s eligibility changes from one cohort to the next, results will default according to the school's latest eligibility. For example, if a school was eligible for funding in the 2025 cohort but becomes ineligible for funding in 2026, the API will not show the school in the 2026 cohort. 
+
+### View a specific school 
+
+``` 
+GET /schools/{id}?filter[cohort]={year} 
+``` 
+
+This endpoint shows details including the programme type choice and confirmed partnerships for a single school in a given cohort.  
+
+Requests must include:  
+ 
+* `cohort` 
+
+For more detailed technical information, view the `GET /schools/{id}` [endpoint Swagger API documentation](/api/docs/v3#/Schools/get_api_v3_schools__id_). 
+
+### Find delivery partner IDs 
+
+``` 
+GET /delivery-partners 
```

Comment:
I think we should define here that this shows Delivery Partners that we have been told contractually Lead providers can work with for a particular cohort. It's the ones they can work with in partnership with a school.